### ISSUE TITLE
Imprecise detection of public-domain using public-domain_124.RULE

### ISSUE THREAD

**Issue by DennisClark** [nexB/scancode-toolkit#3361]
In a recent scan of ThreadLocalRandom.java in package netty-common-4.1.66.Final-sources.jar the scan reported
public-domain (Licenseref-scancode-public-domain) on line 19 using public-domain_124.RULE 
and 
cc0-1.0 on line 20 using cc0-1.0_93.RULE

An examination of lines 18-20 in the original file indicates that the first detection of public-domain is redundant, misleading, and not really correct since the obvious intention is to declare cc0-1.0. 

lines 18-20: 
` * Written by Doug Lea with assistance from members of JCP JSR-166`
` * Expert Group and released to the public domain, as explained at`
` * https://creativecommons.org/publicdomain/zero/1.0/`

In summary, the correct license detection would be simply cc0-1.0 and nothing else.

**Comment by DennisClark** [comment]
suggestion: a possible solution would be a new RULE that would point the text 

`released to the public domain, as explained at https://creativecommons.org/publicdomain/zero/1.0/`

to cc0-1.0

**Comment by DennisClark** [comment]
scan results attached

[netty-common-4.1.66.Final-sources.jar-v32rc4results.json.zip](https://github.com/nexB/scancode-toolkit/files/11336937/netty-common-4.1.66.Final-sources.jar-v32rc4results.json.zip)
